build.gradle

```diff
@@ -318,7 +318,7 @@ if (project.hasProperty('javaLinkageArtifactIds')) {
     dependsOn project.getTasksByName('publishMavenJavaPublicationToMavenLocal', true /* recursively */)
     classpath = project.configurations.linkageCheckerJava
     main = 'com.google.cloud.tools.opensource.classpath.LinkageCheckerMain'
-    args '-a', project.javaLinkageArtifactIds.split(',').collect({"${project.ext.mavenGroupId}:${it}:${project.version}"}).join(',')
+    args '-a', project.javaLinkageArtifactIds.split(',').collect({"${project.ext.mavenGroupId}:${it}:0.1"}).join(',')
```

Comment:
You don't want to make this change permanent.

You could check to see if the artifact id has a `:` and if so skip appending the version. Then people could specify `-PjavaLinkageArtifactIds=beam-vendor-grpc-1_26_0:0.1-SNAPSHOT` and then we would get linkage results `org.apache.beam:beam-vendor-grpc-1_26_0:0.1-SNAPSHOT`